.github/workflows/docker-image.yml

```diff
@@ -14,6 +14,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ${{ github.workspace }}/flask-api/requirements.txt
```

Comment:
So because were running pylint on the runner we do need to pip install the requirements or it will give an error for the modules were importing I belive. I could however get rid of this and execute the pylint on the docker. So right now the workflow order is:

Pylint -> Build -> Test

I would have to change it to:
Build -> Pylint -> Test  (you could also test then run pylint)

but that way everthing is in the docker I guess and we only pip install things once

HOWEVER: Note that now our docker (so our test enviroment) is going to have a extra modules it doesnt need (pylint)

Reply:
yeah might be a good idea to run it on docker. forgot about the import linting